cmw-dto/src/main/java/com/liberologico/cloudesire/cmw/model/dto/PageRequestDTO.java

```diff
@@ -46,7 +46,7 @@ public PageRequestDTO( int pageNumber, int pageSize, String sortField, SortDirec
         this.sortDirection = sortDirection;
     }
 
-    public class Fields
+    public class PaginatorQueryParam
```

Comment:
let's call this inner class just `Parameters` so it gets called as

    PageRequestDTO.Parameters.PAGE_NUMBER